### ISSUE TITLE
Review and discuss 508 assessment responses

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/nrrd#3903]

**Comment by cthomasONRR** [comment]
We followed up with Sid on 5/19-

- DOI will submit manual testing conformance info. 
    - I emailed Sid our testing results for them to submit, so they won't have to test ONRR's content themselves.
- Sid confirmed that one of us will submit the ~40 question responses on OMB Collect. 
    - He is sending OMB my email to get us access. 
    - OMB Connect should open June 1.

**Comment by cthomasONRR** [comment]
Teams conversation with Sid on 5/20-

> Hi Lindsay and Christine, per the [Annual Section 508 Assessment homepage on MAX](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcommunity-dc.max.gov%2Fx%2FYA1vmw&data=05%7C02%7CSiddhartha_Sharma%40ios.doi.gov%7C5fe42810990b4185eac708dd882daa51%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C638816451898442826%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=SIVEhaqiECHEfxYykc2mGOm0%2BXxwmEjtJzfFCylFX0c%3D&reserved=0), OMB will use the list of departments, agencies, and bureaus from [Circular A-11 Appendix C](https://www.whitehouse.gov/wp-content/uploads/2018/06/a11.pdf#page=849) to structure this data collection.  ONRR is not identified in Appendix C of OMB Circular No. A–11. Please confirm - the document is not fully accessible to my screen reader. If that is the case, it is my understanding that you are not required to answer the questions in the "Components" tab. I'll include your test results in the Department's response.

I confirmed that Office of the Secretary/ONRR is not on Appendix C. But, there is a "Departmental Offices" line item that we might fall under. Sid is checking in with his Budget contacts to see if we should be listed in this Appendix and if we should submit question responses. I told Sid that we already answered the questions and can send them to him to have on file even if we don't have to submit to OMB.
